modules/mono/editor/GodotTools/GodotTools.IdeMessaging/Client.cs

```diff
@@ -343,10 +343,10 @@ public async Task<TResponse> SendRequest<TResponse>(Request request)
             }
 
             string body = JsonConvert.SerializeObject(request);
-            return await peer.SendRequest<TResponse>(request.Id, body);
+            return await peer!.SendRequest<TResponse>(request.Id, body);
```

Comment:
`IsConnected` implies that `peer` is not null so we should annotate `IsConnected` instead of using the null forgiveness operator here.

```csharp
[MemberNotNull(nameof(peer))]
public bool IsConnected => ...
```